astro/src/content/quickstarts/_run-fusionauth-other.mdx

```diff
@@ -0,0 +1,6 @@
+If you plan on running FusionAuth another way, check out our [downloads page](/download). When you start FusionAuth for the first time, go through the First-time Setup wizard - you'll see it at the top of the main Dashboard page. At the end of First-time Setup, you'll find the information you need for the quickstart configuration on the Summary page at the end of the wizard. You'll need to:
+1. Take those values and use them in the `.env` config file
+2. Update your application in FusionAuth
+    1. Go to the Applications area in FusionAuth and edit your application
+    2. On the OAuth tab, set the `Authorized Redirect URLs` to `http://localhost:5000/callback`
```

Comment:
We will want to send in the URL to this component so that it matches the rest of the quickstart.